packages/salesforce/src/Adaptor.js

```diff
@@ -879,6 +738,37 @@ export function retrieve(sObjectName, id) {
   };
 }
 
+export function salesforceRequest(path, request) {
```

Comment:
I think this guy needs moving into http now? It's not otherwise referenced in Adaptor.js, and it has all the logic required by http.js. So I think we can just slide it across and declare victory

Reply:
From our sync: We export `sfRequest() => conn.request()` and imported it in `http.js`